lib/ecto/migration/runner.ex

```diff
@@ -216,6 +216,8 @@ defmodule Ecto.Migration.Runner do
     end
   end
 
+  defp reverse(func) when is_function(func, 0), do: func
```

Comment:
Per comment below, I think this clause is wrong, we should remove it.

Reply:
That's right, it was based on `reverse` for old `dynamic` proposal and now it should not work like that any more.